subgraph/schema.graphql

```diff
@@ -0,0 +1,73 @@
+type Token @entity {
+  id: ID!
+  creator: Creator! # address
+  name: String! # string
+  symbol: String!
+  totalSupply: BigInt!
+  decimals: Int
+  creatorTokenRatio: Int
+  isTotalSupplyFixed: Boolean
+  lockupPeriod: Int
+  enableStakeToToken: Boolean
```

Comment:
I thought `enableStakeToToken` was also unnecessary because we don't currently plan on staking.